egs2/dns_ins20/enh1/README.md

```diff
@@ -14,9 +14,29 @@
  - config: ./conf/tuning/train_enh_blstm_tf.yaml
  - Pretrained model: https://zenodo.org/record/4923697
 
-|dataset|STOI|SAR|SDR|SIR|
-|---|---|---|---|---|
-|enhanced_cv_synthetic|0.95|18.63|18.63|0.00|
-|enhanced_tt_synthetic_no_reverb|0.92|10.92|10.92|0.00|
-|enhanced_tt_synthetic_with_reverb|0.85|9.31|9.31|0.00|
+| dataset                           | STOI | SAR   | SDR   | SIR  |
+| --------------------------------- | ---- | ----- | ----- | ---- |
+| enhanced_cv_synthetic             | 0.95 | 18.63 | 18.63 | 0.00 |
+| enhanced_tt_synthetic_no_reverb   | 0.92 | 10.92 | 10.92 | 0.00 |
+| enhanced_tt_synthetic_with_reverb | 0.85 | 9.31  | 9.31  | 0.00 |
 
+<!-- Generated by ./scripts/utils/show_enh_score.sh -->
+# RESULTS
+## Environments
+- date: `Thu Feb 10 23:11:40 CST 2022`
+- python version: `3.8.12 (default, Oct 12 2021, 13:49:34)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.5a1`
+- pytorch version: `pytorch 1.9.1`
+- Git hash: `6f66283b9eed7b0d5e5643feb18d8f60118a4afc`
+  - Commit date: `Mon Dec 13 15:30:29 2021 +0800`
+
```

Comment:
Can you upload a model to the HF hub and add a link here?

Reply:
Ok, I have uploaded the model to the HF hub, and the link is https://huggingface.co/Johnson-Lsx/Shaoxiong_Lin_dns_ins20_enh_enh_train_enh_dccrn_raw